src/vorta/scheduler.py

```diff
@@ -246,17 +246,19 @@ def set_timer_for_profile(self, profile_id: int):
 
             # calculate next scheduled time
             if profile.schedule_mode == 'interval':
-                last_time = last_run_log.end_time
+                last_time: dt = last_run_log.end_time
```

Comment:
Is this a type hint? Didn't know one would put them inside functions.

Reply:
It is. You can use them with every variable but often there is no need to in case of local variable since linters will be able to recognise the time on their own. In this case however the linter doesn't know the type of `last_run_log.end_time` since peewee is not typed.